### PR TITLE
Video overview

### PR DESCRIPTION
**Pull request by colorfield**
Site building, backend/frontend dev for #6 
As there is already a lot going on with this PR, created proposed follow ups in the original issue.

### CODE REVIEW

**Comment by colorfield** [comment]
@dan2k3k4 while moving to search api, these conditions have been adapted https://github.com/drupal-celebrations/celebrate-drupal-9/pull/29/commits/395900524d1fd0c238da83da072b03936f7029ed#diff-994789ec470bc8c7745ba5ef2d8a3061R40-R42 but i might miss other classes for the columns, it would be great if you could have a look. Thanks!

File: web/modules/custom/user_registration/images/drupal-evergreen-logo.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 42.2 55.5">
```
Comment (dan2k3k4) [inline web/modules/custom/user_registration/images/drupal-evergreen-logo.svg]: As we noticed, we'd need to tweak this to full numbers like: `viewBox="0 0 44 58"`

Comment (colorfield) [reply web/modules/custom/user_registration/images/drupal-evergreen-logo.svg]: Thanks! Oddly here it does not seem to have any influence, perhaps it was related to the path.
Screenshot 2020-05-24 at 01 22 36

Comment (colorfield) [reply web/modules/custom/user_registration/images/drupal-evergreen-logo.svg]: Still rounded the viewBox https://github.com/drupal-celebrations/celebrate-drupal-9/pull/29/commits/ec5017984e6c334ff9c3450c8894f9d770267088

**Review by dan2k3k4 — APPROVED** [review]
Some nitpicks like remove `field_cc_license` but to save time I'll get this merged and we can remove things we don't need after

**Pull request merged**